### ISSUE TITLE
Docker warnings “You have opted to share your home directory with a container”

### ISSUE THREAD

**Issue by dalin-** [lando/lando#3472]
Today I thought “Docker’s been prodding me to upgrade for a while, I guess I’ll finally give in”.  So I upgraded to 4.14.1.  Now every time I start a lando site I get 6 popups saying “You have opted to share your home directory with a container.”  and then details about how this is bad for performance and possibly exposes your private stuff.

Is there some secret way to speed up Lando by stopping sharing my home directory?

Some containers might need it to be able to, for example share keys.  But my guess is that most containers don't (e.g. DB, Solr, edge, mailhog, phpmyadmin).  

If I get a chance I'll do some experimentation.

**Comment by renebakx** [comment]
That limit is implied by the docker engine, not Lando. 

What I did was specify directories that it was allowed to use instead of bluntly giving access to the entire ~/ 


![image](https://user-images.githubusercontent.com/347093/204243946-d56be406-cdc3-47a1-a752-0e2b0ef1ea3b.png)

With some experimenting I got it boiled down the above. Not perfect but it works. Please note I do not need my ~/.ssh/ to be used with Docker at all.

**Comment by dalin-** [comment]
@renebakx 's suggestion didn't work for me. I made those changes, and then

```
lando destroy -y
...
lando start
...
Creating bcj_appserver_1 ... error

ERROR: for bcj_appserver_1  Cannot start service appserver: Mounts denied:
The path /Users/dave is not shared from the host and is not known to Docker.
You can configure shared paths from Docker -> Preferences... -> Resources -> File Sharing.
See https://docs.docker.com/desktop/mac for more info.
```

**Comment by kristjankoppel** [comment]
@dalin- did you manage to find solution to this problem?

**Comment by dalin-** [comment]
@kristjankoppel 

> did you manage to find solution to this problem?

No.  Eventually I just clicked the button "Don't warn me again".

**Comment by yaronuliel** [comment]
Any chance someone can officially address this issue?
It makes no sense that a tool that is built on top of docker will require settings that are considered not recommended.

What we need is a list of the specific directories within the  home directory that lando requires in order to fully work,
I know for example that the `~/.ssh` directory is one of them (at least with default openssh configuration), there is also `~/.lando` and `~/.composer` that was mentioned by @renebakx, anything else?

**Comment by danepowell** [comment]
https://github.com/lando/lando/issues/2635 is an existing feature request to not mount the home directory. I think we could combine this issues.

There's also a good suggested workaround: https://github.com/lando/lando/issues/2635#issuecomment-877473886

In your ~/.lando/config.yml you can set home to a empty string to not mount the user folder.
E.g.

```
home: ''
```
